app/models/expanded_taxonomy.rb

```diff
@@ -0,0 +1,47 @@
+class ExpandedTaxonomy
```

Comment:
Ok, that rename makes things better (the naming got me a bit confused).

I tried to use Taxon objects instead of the hashes and ended up building a small Tree class to replace rubytree. https://github.com/alphagov/content-tagger/commit/8f6099e04ce9dc962063712545bbdbe32755670d If there is something useful there feel free to use, otherwise the renaming is fine.